cmd/ci-operator-prowgen/main.go

```diff
@@ -79,9 +79,9 @@ func generatePodSpec(org, repo, configFile, target string, additionalArgs ...str
 	configMapKeyRef := kubeapi.EnvVarSource{
 		ConfigMapKeyRef: &kubeapi.ConfigMapKeySelector{
 			LocalObjectReference: kubeapi.LocalObjectReference{
-				Name: fmt.Sprintf("ci-operator-%s-%s", org, repo),
+				Name: "ci-operator-configs",
 			},
-			Key: configFile,
+			Key: org + "-" + repo + "-" + configFile[0:len(configFile)-len(filepath.Ext(configFile))],
```

Comment:
Pass the branch name as an argument instead of re-parsing the file name here.

Reply:
the branch name is the filename

Reply:
I am not parsing it, I am just removing the extension here (`.yaml`)

Reply:
Consider using `Sprintf` and `TrimSuffix` (https://golang.org/pkg/strings/#TrimSuffix) for readability